src/geocoders/here.js

```diff
@@ -28,8 +29,14 @@ export var HERE = L.Class.extend({
   },
 
   reverse: function(location, scale, cb, context) {
+    var _proxRadius = this.options.reverseGeocodeProxRadius
+      ? Number.isInteger(this.options.reverseGeocodeProxRadius)
```

Comment:
This breaks IE compatibility, since IE does not have `isInteger`. Also, is it strictly necessary to check for it? We rarely if ever validate options like this.